client/src/components/RatingsAndReviews/AddReview.jsx

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-unused-vars */
```

Comment:
Should delete any unused variables instead of using eslint-disable no-unused-vars